my-diploma/src/Components/LogOut/LogOut.js

```diff
@@ -0,0 +1,22 @@
+import { useNavigate } from "react-router-dom";
+import ROUTE_NAMES from "../../router/routeName";
+import { useCallback } from "react";
+import styles from "./index.module.scss";
+import Exit from "../../static/exit.png";
+
+const LogOut = () => {
+  const logOut = useCallback(() => {
```

Comment:
1. useCallback is redundant here
2. Better to create separate function/hook for this logic cause of it's could be helpful to run logout not only here